osidb_bindings/templates_0.22.0/client.py.jinja

```diff
@@ -0,0 +1,238 @@
+{#
+    This is a custom template derived from:
+    https://github.com/openapi-generators/openapi-python-client/tree/v0.22.0/openapi_python_client/templates/client.py.jinja
+#}
+{# CHANGE START (1) - use old style client from openapi-python-client 0.10.7 #}
+{# import ssl
```

Comment:
We are keeping the new template commented and still using the old implementation so we can change it later right?